src/test/java/michael/StorageTest.java

```diff
@@ -0,0 +1,21 @@
+package michael;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StorageTest {
```

Comment:
Do consider more tests for all the methods inside Storage and testing all the scenarios for each method.